Lib/test/test_typing.py

```diff
@@ -4228,6 +4248,9 @@ def test_get_type_hints(self):
             {'a': typing.Optional[int], 'b': int}
         )
 
+    def test_get_type_hints_generic(self):
+        assert set(get_type_hints(BarGeneric)) == {'a', 'b'}
```

Comment:
Could you also add tests to verify generic `TypedDict` bases?

And it will be great to have tests for generic `TypedDict` subscription.

Reply:
On it